.travis.yml

```diff
@@ -44,3 +44,13 @@ script:
   - npm run jscs-enforce
   - npm run checkdb
   - npm run test-travis
+
+before_install:
+ - npm install -g npm@^5.7.0
```

Comment:
line 49 should have two spaces before the dash
line 55 should have two spaces before the word "directories"
line 56 should have four spaces before the dash

Reply:
did you generate the package lock file *after* pulling all the recent module updates?
did you run `npm update` *before* generating the package lock file?